packages/core/README.md

```diff
@@ -78,117 +62,378 @@ app.all("/mcp", async (c) => {
 });
 ```
 
-## Creating an MCP Server
+> [!TIP]
+>
+> The Model Context Protocol (MCP) is an open standard that enables secure connections between host applications and external data sources and tools, allowing AI assistants to reason over information and execute functions with user permission.
 
-Basic constructor usage:
+## Features
 
-```typescript
-import { McpServer } from "mcp-lite";
+- No runtime dependencies and a single TypeScript entrypoint.
+- Type-safe tool definitions with Standard Schema (Zod, Valibot, Effect, ArkType).
+- Structured outputs with runtime validation and schema exposure via `tools/list`.
+- HTTP + SSE transport built on the Fetch API (no stdio wrapper required).
+- Adapter interfaces for sessions, client requests, and persistence when you outgrow stateless mode.
+- Middleware hooks and server composition via `.group()` for modular setups.
 
-const server = new McpServer({
-  name: "my-server",
-  version: "1.0.0",
-});
+## Installation
```

Comment:
i like when installation is towards the top of the fold, but that's personal preference